kylin-spark-project/kylin-spark-common/src/test/java/org/apache/spark/sql/common/SparkQueryTest.scala

```diff
@@ -116,9 +116,9 @@ object SparkQueryTest {
                    |== Results ==
                    |${
                     sideBySide(
-                        s"== Kylin Answer - ${kylinAnswer.size} ==" +:
+                        s"== Expected Answer - ${kylinAnswer.size} ==" +:
```

Comment:
Is it correct ? I think `sparkResults` should be the expected result.

Reply:
yes, it's correct, the 'sparkResults' is the results querying from sparder, and 'kylinResults' is the expected results.